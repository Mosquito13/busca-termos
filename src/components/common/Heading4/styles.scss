.heading-4 {
    font-size: 2rem;
    font-weight: 400;
    color: currentColor;
}