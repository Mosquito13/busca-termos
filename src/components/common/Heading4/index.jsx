import './styles.scss';

const Heading4 = ({ children }) => {
  return <h4 className='heading-4'>{children}</h4>;
};

export default Heading4;
