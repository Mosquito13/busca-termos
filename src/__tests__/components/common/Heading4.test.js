import renderer from 'react-test-renderer';

import Heading4 from '../../../components/common/Heading4';

describe(`${Heading4.name}`, () => {
  describe('snapshots', () => {
    it('should render', () => {
      const tree = renderer
        .create(<Heading4 value="value for heading" />)
        .toJSON();

      expect(tree).toMatchSnapshot();
    });
  });
});
