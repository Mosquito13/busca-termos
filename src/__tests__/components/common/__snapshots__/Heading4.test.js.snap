// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading4 snapshots should render 1`] = `
<h4
  className="heading-4"
>
  value for heading
</h4>
`;
